src/coreclr/vm/codeman.cpp

```diff
@@ -6254,6 +6332,15 @@ BOOL ReadyToRunJitManager::JitCodeToMethodInfo(RangeSection * pRangeSection,
     // Save the raw entry
     PTR_RUNTIME_FUNCTION RawFunctionEntry = pRuntimeFunctions + MethodIndex;
 
+    ULONG UMethodIndex = (ULONG)MethodIndex;
+
+    const int lookupIndex = HotColdMappingLookupTable::LookupMappingForMethod(pInfo, (ULONG)MethodIndex);
+    if ((lookupIndex != -1) && ((lookupIndex % 2) == 1))
```

Comment:
Nit - modulus needs a more complex assembly sequence for signed ints, could we just test the zero bit using <code>lookupIndex &amp; 1</code>? I think that in this particular case it would be even more expressive w.r.t. intent of the check.